meshtastic/tcp_interface.py

```diff
@@ -30,10 +30,10 @@ def __init__(
         self.portNumber = portNumber
 
         if connectNow:
-            logging.debug(f"Connecting to {hostname}")
+            logging.debug(f"Connecting to {hostname}") # type: ignore[str-bytes-safe]
             server_address = (hostname, portNumber)
             sock = socket.create_connection(server_address)
-            self.socket = sock
+            self.socket: Optional[socket.socket] = sock
```

Comment:
This made mypy happier since this could be set to None as well as being an actual socket.